mne/viz/backends/renderer.py

```diff
@@ -36,18 +36,13 @@ def set_3d_backend(backend_name):
 
     Parameters
     ----------
-    backend_name : {'mlab'}, default is 'mlab'
+    backend_name : str
+        The 3d backend to select.
     """
-    _validate_type(backend_name, "str")
-
+    _check_option('backend_name', backend_name, VALID_3D_BACKENDS)
     global MNE_3D_BACKEND
-    if backend_name == 'mayavi':
-        MNE_3D_BACKEND = backend_name
-    else:
-        raise ValueError('backend_name should be "mayavi" : '
-                         '{} was given.'.format(MNE_3D_BACKEND))
-    from . import renderer
-    importlib.reload(renderer)
+    MNE_3D_BACKEND = backend_name
+    importlib.reload(sys.modules[__name__])
```

Comment:
The LGTM warning was still bothering me so I found a way to avoid the import (I think). Hopefully you won't need to revert it when you actually go to switch backends.